src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -222,6 +223,40 @@ export class TabsTitleControl extends TitleControl {
 		});
 	}
 
+	private updateTabSizing(): void {
+		if (this.tabsContainer) {
+			const options = this.accessor.partOptions;
+			if (options.tabSizing === 'fixed') {
+				this.tabsContainer.style.setProperty('--tab-sizing-max-width', options.tabSizingMaxWidth + 'px');
+				this.tabsContainer.onmouseenter = () => this.fixTabWidths();
+				this.tabsContainer.onmouseleave = () => this.relaxTabWidths();
+			} else {
+				this.tabsContainer.style.removeProperty('--tab-sizing-max-width');
+				this.tabsContainer.onmouseenter = null;
+				this.tabsContainer.onmouseleave = null;
+				this.relaxTabWidths();
+			}
+		}
+	}
+
+	private fixTabWidths() {
+		if (this.accessor.partOptions.tabSizing === 'fixed') {
+			this.forEachTab((editor, index, tabContainer, tabLabelWidget, tabLabel, tabActionBar) => {
+				const { width } = tabContainer.getBoundingClientRect();
+				tabContainer.style.setProperty('--tab-sizing-current-width', width.toFixed(2) + 'px');
```

Comment:
```suggestion
				tabContainer.style.setProperty('--tab-sizing-current-width', `${width.toFixed(2)}px`);
```

Reply:
Also, what is the purpose of using `toFixed` here?

Reply:
toFixed came from the Atom patch, it doesn't seem useful, so I'll remove it